src/types/profile.js

```diff
@@ -184,6 +191,7 @@ export type ProfileMeta = {|
   oscpu: string,
   platform: string,
   processType: number, // TODO find the possible values
+  extensions?: ExtensionTable,
```

Comment:
Could you add the comment.

```
// The extensions property landed in Firefox 60, and is only optional because older processed profile versions
// may not have it. No upgrader was written for this change.
```